app/components/pagy_paginator_component/bottom_component.html.erb

```diff
@@ -0,0 +1,25 @@
+<div id="<%= @id %>" class="<%= @container_classes %>">
+  <%= content %>
+</div>
+
+<div class="mt-8">
+  <%= pagy_links do %>
+    <%== pagy_nav @pagy %>
+  <% end %>
+</div>
+
+<%= tag.turbo_frame id: "#{@id}-#{@pagy.page}" do %>
```

Comment:
Just wanted to point out the weird verbiage here. There are [known](https://github.com/github/view_component/issues/1099) issues with using `turbo_frame` and `turbo_stream` inside view components. So the only option to be sure it works currently is to manually build the tags